blues/posedart.py

```diff
@@ -393,10 +368,10 @@ def dartmove(self, context=None, residueList=None):
         selectedboard, changevec = self.calc_from_center(com=center)
         print('changevec', changevec)
         if selectedboard != None:
-        #notes
-        #comMove is where the com ends up after accounting from where
-        #it was from the original dart center
-        #basically it's final displacement location
+            #notes
```

Comment:
I prefer not to have "notes" here. It could be worth saying this in comments here, but having "notes" makes it look like we've left notes to ourselves in the code.

(I realize it was already there. Just remarking...)

Reply:
(Also if you update, perhaps also update to explain "com").

Reply:
I've excluded smartdart/posedart code from tests and code coverage since I do not fully understand how these work since this was written by Sam. So I will leave updating these pieces of code to Sam.